tests/Authorization/AuthorizableTest.php

```diff
@@ -295,4 +296,28 @@ public function testCanCascadesToGroupsWithWildcards(): void
 
         $this->assertTrue($this->user->can('admin.access'));
     }
+
+    /**
+     * @see https://github.com/codeigniter4/shield/pull/238
+     */
+    public function testCreatedAtIfDefaultLocaleSetFaWithAddGroup(): void
+    {
+        $currentLocale = Locale::getDefault();
+        Locale::setDefault('fa');
+
+        $this->hasInDatabase('auth_groups_users', [
+            'user_id'    => $this->user->id,
+            'group'      => 'admin',
+            'created_at' => Time::now()->format('Y-m-d H:i:s'),
+        ]);
+
+        $this->user->addGroup('admin');
+
+        $this->dontSeeInDatabase('auth_groups_users', [
+            'user_id'    => $this->user->id,
+            'group'      => 'admin',
+            'created_at' => '0000-00-00 00:00:00',
```

Comment:
Isn't `'0000-00-00 00:00:00'` MySQL specific?

Does this test fail before applying your fixes?